scripts/generate_prefrosh.py

```diff
@@ -0,0 +1,18 @@
+import sqlalchemy
+import os
+from ruddock import config
+
+engine = sqlalchemy.create_engine(config.DEV.db_uri, convert_unicode=True)
+
+if __name__ == "__main__":
+  for filename in os.listdir('../../images/'):
```

Comment:
Don't think you should hardcode this. Maybe you should pass in the path to the image directory as a command line arg? Then running this script would look like this:
```
python import_prefrosh.py --env dev /path/to/image/dir/
```